docusaurus.config.ts

```diff
@@ -72,7 +72,7 @@ const config: Config = {
     },
     colorMode: {
       defaultMode: "light",
-      disableSwitch: true,
+      disableSwitch: false,
```

Comment:
Although this is a draft PR, I think we should not enable the switch to change modes.